test/interval_tests/linear_algebra.jl

```diff
@@ -1,23 +1,17 @@
 using IntervalArithmetic
 using Test
 
-
-
 A = [ 2..4   -2..1
      -1..2    2..4]
 
 b = [-2..2
      -2..2]
 
-
 @testset "Linear algebra with intervals tests" begin
-
-    @test A * b == [-12..12
-                    -12..12]
+    @test all(A * b .≛ [-12..12, -12..12])
 
     # Example from Moore et al., Introduction to Interval Analysis (2009), pg. 88:
-
-    @test A \ b == [-5..5
-                    -4..4]
-
+    # TODO This is using the wrong \. I think the correct one is defined
```

Comment:
I think this should be removed (also it would error if the ternary logic defined here is used).

A few curiosities if you are interested: The `\` defined in IntervalRootFinding preconditions by the approximate inverse midpoint. Preconditioning an interval linear system actually enlarges the solution set (fun fact, the solution set of this is the logo of IntervalLinearAlgebra.jl), so the backslash overloaded there would return [-14..14, -14..14], which is the interval hull of the preconditioned solution set.